lexos/static/js/scripts_cut.js

```diff
@@ -1,382 +1,663 @@
+        // For cutting by characters
+        if (defCutTypeValue === 'letters') {
+            // Check each document without individual options
+            eltswithoutindividualopts.forEach(function (elt) {
+                // Needs warning...
+                // If the number of characters-segment size/segment size-overlap size > 100
+                if ((numChar[elt] - cutVal) / (cutVal - overVal) > maxSegs) {
+                    return true
+                }
+            })
+            // Do the same with words and lines
         }
-      }
-    }
-  })
-
-  // If cut by milestone is checked
-  if ($("input[name='cutByMS']:checked").length == 0) {
-    // For cutting by characters
-    if (defCutTypeValue == 'letters') {
-      // Check each document without individual options
-      eltswithoutindividualopts.forEach(function (elt) {
-        // Needs warning...
-        // If the number of characters-segment size/segment size-overlap size > 100
-        if ((numChar[elt] - cutVal) / (cutVal - overVal) > maxSegs) {
-          needsWarning = true
+        else if (defCutTypeValue === 'words') {
+            eltswithoutindividualopts.forEach(function (elt) {
+                if ((numWord[elt] - cutVal) / (cutVal - overVal) > maxSegs) {
+                    return true
+                }
+            })
         }
-      })
-      // Do the same with words and lines
-    } else if (defCutTypeValue == 'words') {
-      eltswithoutindividualopts.forEach(function (elt) {
-        if ((numWord[elt] - cutVal) / (cutVal - overVal) > maxSegs) {
-          needsWarning = true
+        else if (defCutTypeValue === 'lines') {
+            eltswithoutindividualopts.forEach(function (elt) {
+                if ((numLine[elt] - cutVal) / (cutVal - overVal) > maxSegs) {
+                    return true
+                }
+            })
+            // If the segment size > 100 and there are documents without individual options
         }
-      })
-    } else if (defCutTypeValue == 'lines') {
-      eltswithoutindividualopts.forEach(function (elt) {
-        if ((numLine[elt] - cutVal) / (cutVal - overVal) > maxSegs) {
-          needsWarning = true
+        else if (cutVal > maxSegs && eltswithoutindividualopts.length > 0) {
+            return true
         }
-      })
-      // If the segment size > 100 and there are documents without individual options
-    } else if (cutVal > maxSegs && eltswithoutindividualopts.length > 0) {
-      needsWarning = true
+    } // end if
+    return false
+} // end checkMilestone
+
+
+// checkIndividualCutset
+/**
+ * checks each individual cutset for warnings
+ * @param {number} maxSegs - maximum number of segments
+ * @param {string} defCutTypeValue - cut type
+ * @param {number} cutVal - segment size
+ * @param {number} overVal - overlap size
+ * @param {array} indivdivs - all individual cutsets
+ * @param {array} eltswithoutindividualopts - elements without cutsets
```

Comment:
array of what? see jsdoc type definition: http://usejsdoc.org/tags-type.html for way to annotate array of number, array of string, etc.